src/main/java/com/rebuy/consul/RebuyConsulClient.java

```diff
@@ -0,0 +1,38 @@
+package com.rebuy.consul;
+
+import com.ecwid.consul.v1.ConsulClient;
+import com.ecwid.consul.v1.agent.model.NewService;
+
+import java.util.ArrayList;
+
+public class RebuyConsulClient
+{
+    public final static String CONSUL_ADDRESS = "192.168.33.10";
```

Comment:
Hmm ... can we just use `localhost` here, since there is an agent on every host?